autoload/ale/python.vim

```diff
@@ -27,7 +27,7 @@ endfunction
 function! ale#python#FindVirtualenv(buffer) abort
     for l:path in ale#path#Upwards(expand('#' . a:buffer . ':p:h'))
         for l:dirname in ale#Var(a:buffer, 'virtualenv_dir_names')
-            let l:venv_dir = l:path . '/' . l:dirname
```

Comment:
If you want to convert `//` to `/`, use the `simplify` function. See `:help simplify()`.